src/modules/octopi/start_chroot_script

```diff
@@ -34,21 +34,55 @@ else
 fi
 apt-get -y --force-yes install python3 python3-virtualenv git screen subversion cmake avahi-daemon libavahi-compat-libdnssd1 libatlas3-base
 
+#apt-get -y install software-properties-common
```

Comment:
Please delete commented out code if we don't need it

Reply:
It is already merged 😅
I can remove it if i have a new PR

Reply:
Aaah I didn't notice :rofl: